LambdaEngine/Source/Game/GameConsole.cpp

```diff
@@ -220,6 +221,127 @@ namespace LambdaEngine
 			});
 		}
 
+		// Set Boolean in config
+		{
+			ConsoleCommand cmdSetBool;
+			cmdSetBool.Init("set_bool", true);
+			cmdSetBool.AddArg(Arg::EType::STRING);
+			cmdSetBool.AddArg(Arg::EType::BOOL);
+			cmdSetBool.AddDescription("Change boolean value in config file with a given key\n\t 'set_bool key bool'");
+			BindCommand(cmdSetBool, [this](GameConsole::CallbackInput& input)->void
+				{
+					if (!EngineConfig::SetBoolProperty(input.Arguments.GetFront().Value.String, input.Arguments.GetFront().Value.Boolean))
```

Comment:
Fix the commands. Use argument 0 for the string and argument 1 for bool.
The problem is the same for the other settings commands which change variables.